lib/matplotlib/scale.py

```diff
@@ -478,10 +478,111 @@ def get_transform(self):
         return self._transform
 
 
+def _mask_non_logit(a):
+    """
+    Return a Numpy masked array where all values outside ]0, 1[ are
```

Comment:
This docstring does not match what the function is doing.  It is always returning a copy of the input array in which any values outside ]0, 1[ are replaced by nan.